composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
@dunglas

> and they can still disable the "hijacking" using the config if they want to

That should be in the documentation. Preferably include a link from the above error message too.